skietbaan_fe/src/bootstrap/LeaderboardStyle.css

```diff
@@ -2,52 +2,99 @@
     min-width: 100%;
 }
    
-.filterNav{
-    min-width: 100%;
-    display: inline-flex;
-    background-color: darkgrey;
-    padding: 1%;
+.filterSection{
+    background-color: #171F32;
+    color: white;
+    width: 100%;
 }
-.competitionSelction{
+.filterTableAlwaysVisible{
     width:100%;
-    background-color: black;
+    margin:3%;
+}
+.filterContent{
+    background-color: #171F32;
+    color: white;
+    width: 100%;
+    padding-left: 3%;
+    padding-right: 3%;
+    align-content: center;
+}
+.row div p{
+    color: #FF611D;
+}
+.row div .scrollableContainerC{
+    overflow-y: auto;
+    height: 150px;
```

Comment:
px? sorry don't know context

Reply:
pixel, so only three rows will be visible at a time in that table